.storybook/docgen-plugin.js

```diff
@@ -8,8 +8,8 @@ const tsconfigPath = path.join(__dirname, '../tsconfig.json');
 const fs = require('fs');
 
 const propFilter = (fileName, prop, component) => {
-  // `PropTables.tsx` files are okay to pass through
-  if (fileName.includes('PropTables.tsx')) {
+  // `PropTables.splitprops.tsx` files are okay to pass through
+  if (fileName.includes('PropTables.splitprops.tsx')) {
```

Comment:
I'm reverting this change since it is in fact cluttering up existing prop tables (see the enormous prop tables for the [`Text`](https://5d854c26ba934e0020f5e98a-qqxsxgwqqm.chromatic.com/?path=/docs/preview-text-react--basic-text) component). We'll save the docgen fixes for a separate PR -- `Grid` isn't the only component with incomplete prop tables (see `Box`, `Flex`, `Stack`), we should be able to fix most of them in a single PR.